netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -0,0 +1,215 @@
+                {
+                    targetPathToItem[targetPath] = item;
+                }
+            }
+        }
+
+
+        private static string GetSourcePath(ITaskItem item)
+        {
+            var sourcePath = item.GetMetadata("HintPath");
+
+            if (String.IsNullOrWhiteSpace(sourcePath))
+            {
+                sourcePath = item.ItemSpec;
+            }
+
+            return sourcePath;
+        }
+
+        static readonly string[] s_targetPathMetadata = new[] { "TargetPath", "Path" };
+        private static string GetTargetPath(ITaskItem item)
+        {
+            foreach (var metadata in s_targetPathMetadata)
+            {
+                var targetPath = item.GetMetadata("TargetPath");
+
+                if (!String.IsNullOrWhiteSpace(targetPath))
+                {
+                    return targetPath.Replace('\\', '/');
+                }
+            }
+
+            var sourcePath = GetSourcePath(item);
+
+            return Path.GetFileName(sourcePath);
+        }
+
+
+        private ITaskItem HandleConflict(ITaskItem item1, ITaskItem item2)
+        {
+            var conflictMessage = $"Encountered conflict between {item1.ItemSpec} and {item2.ItemSpec}.";
+
+            var sourcePath1 = GetSourcePath(item1);
+            var sourcePath2 = GetSourcePath(item2);
+
+            var exists1 = File.Exists(sourcePath1);
+            var exists2 = File.Exists(sourcePath2);
+
+            if (!exists1 && !exists2)
+            {
+                Log.LogMessage($"{conflictMessage}.  Could not determine winner because {sourcePath1} and {sourcePath2} do not exist.  Arbitrarily choosing {item1.ItemSpec}.");
+                return item1;
+            }
+
+            if (exists1 && !exists2)
+            {
+                Log.LogMessage($"{conflictMessage}.  Choosing {item1.ItemSpec} because {sourcePath1} exists and {sourcePath2} does not.");
+                return item1;
+            }
+
+            if (!exists1 && exists2)
+            {
+                Log.LogMessage($"{conflictMessage}.  Choosing {item2.ItemSpec} because {sourcePath2} exists and {sourcePath1} does not.");
+                return item2;
+            }
+
+            var assemblyVersion1 = GetAssemblyVersion(sourcePath1);
+            var assemblyVersion2 = GetAssemblyVersion(sourcePath2);
+
+            if (assemblyVersion1 != null || assemblyVersion2 != null)
+            {
+                if (assemblyVersion1 == null)
+                {
+                    Log.LogMessage($"{conflictMessage}.  Choosing {item2.ItemSpec} because {sourcePath1} had no AssemblyVersion and {sourcePath2} did.");
+                    return item2;
+                }
+
+                if (assemblyVersion2 == null)
+                {
+                    Log.LogMessage($"{conflictMessage}.  Choosing {item1.ItemSpec} because {sourcePath2} had no AssemblyVersion and {sourcePath1} did.");
+                    return item1;
+                }
```

Comment:
I think providing warnings if one side is an assembly and the other isn't seems reasonable.